api/v1alpha1/environmentrequest_types.go

```diff
@@ -61,8 +61,10 @@ type EnvironmentRequestSpec struct {
 	// TODO: Dataset per provider?
 	Dataset *apiextensionsv1.JSON `json:"dataset,omitempty"`
 
-	Providers EnvironmentProviders `json:"providers"`
-	Splitter  Splitter             `json:"splitter"`
+	Providers          EnvironmentProviders `json:"providers"`
+	Splitter           Splitter             `json:"splitter"`
+	ServiceAccountName string               `json:"serviceaccountname,omitempty"`
+	SecretRefName      string               `json:"secretrefname,omitempty"`
```

Comment:
How much of this config is actually required by the EnvironmentRequest?

Reply:
I am adding only ServiceAccountName and SecretRefName as optional. 

Can't say about the rest. Any changes there should be done separately.

Reply:
This change only allows starting environment provider jobs without a cluster. 
They will start by the `default` SA, but won't succeed due to permissions. 
To make them succeed will require service accounts not bound to cluster (like `global-provider` instead of `<cluster_name>-provider`).

Reply:
In update [862f006](https://github.com/eiffel-community/etos/pull/320/commits/862f0064f9312b61f2ea30a1caf0acc7cbe7a131) SecretRefName is replaced with JobConfig. The idea is to provide only the required variables.

Reply:
Commit [dc58833](https://github.com/eiffel-community/etos/pull/320/commits/dc5883393b4a88c8d95e3e3d220c30300a25b354) works with 'etosctl testrun start'.

Reply:
Environment requests can be now created without cluster. Verified with `registry.nordix.org/eiffel/etos-environment-provider:d5e65c2f`.
```
$ kubectl create -n etos-ns -f env_request.yaml
...
$ kubectl get environmentrequests -n etos-ns
NAME                    READY   REASON   MESSAGE
quicktest-qhb2m-hkt0b   True    Done     Successfully provisioned an environment
```